tachyon/crypto/commitments/kzg/kzg.h

```diff
@@ -62,7 +63,7 @@ class KZG {
     CHECK_EQ(g1_powers_of_tau_.size(), g1_powers_of_tau_lagrange_.size());
     CHECK_LE(g1_powers_of_tau_.size(), kMaxDegree + 1);
 #if TACHYON_CUDA
-    SetupForGpu();
+    SetupForGpu(g1_powers_of_tau_.size());
```

Comment:
Since `g1_powers_of_tau_` is a member variable, you don't need to pass this as an argument.